CHANGE_LOG

```diff
@@ -18,7 +18,10 @@ Pull-Requests:
 * PR `#726 <https://github.com/numba/llvmlite/pull/726>`_: Disallow alwaysinline and noinline on functions (`Graham Markall <https://github.com/gmarkall>`_)
 * PR `#730 <https://github.com/numba/llvmlite/pull/730>`_: Don't export LLVM symbols when linking statically (`Chris Burr <https://github.com/chrisburr>`_ `esc <https://github.com/esc>`_)
 * PR `#732 <https://github.com/numba/llvmlite/pull/732>`_: Docs/remove travis (`esc <https://github.com/esc>`_)
-* PR `#734 <https://github.com/numba/llvmlite/pull/734>`_: bump llvmdev to 11 (`esc <https://github.com/esc>`_)
```

Comment:
Is this deletion intentional? #734 is on the release branch https://github.com/numba/llvmlite/commits/release0.37 ?

Reply:
looks accidental to me

Reply:
yes, `d` key stuck in vim.